.github/workflows/build-verification.yml

```diff
@@ -60,6 +66,8 @@ jobs:
       fail-fast: false
       matrix:
         versions: ${{ fromJson(needs.generate_versions.outputs.matrix) }}
+        java: [ { jdk: 11, jdkImageIssue: 11.0.12 ,jdkImageIssueAlt:  11.0.13 }, { jdk: 17,  jdkImageIssue:  17.0.3+7 ,jdkImageIssueAlt: 17.0.5+8} ]
```

Comment:
As long as the JDK image issue is tested it with a different minor version, we can consider it covered. I would remove the second test with a different minor version.